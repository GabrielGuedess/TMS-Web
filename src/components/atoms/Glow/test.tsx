import { render } from '@testing-library/react';

import { Glow } from '.';

describe('<Glow />', () => {
  it('should render the glow correctly', () => {
    const { container } = render(<Glow>Test</Glow>);

    expect(container.firstChild).toMatchSnapshot();
  });
});
