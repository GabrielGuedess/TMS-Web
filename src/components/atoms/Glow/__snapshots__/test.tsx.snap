// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Glow /> should render the glow correctly 1`] = `
<div
  class="glow"
  style="display: grid; --glow-top: 0px; --glow-left: 0px;"
>
  <div
    class=""
    style="grid-area: 1/1/1/1;"
  >
    Test
  </div>
  <div
    class="glow-mask "
    glow="true"
    style="--glow-color: #f50057; grid-area: 1/1/1/1; pointer-events: none; mask: radial-gradient(var(--glow-size) var(--glow-size) at calc(var(--glow-x, -99999px) - var(--glow-left, 0px))
calc(var(--glow-y, -99999px) - var(--glow-top, 0px)), #000000 1%, transparent 50%);"
  >
    Test
  </div>
</div>
`;
